### ISSUE TITLE
Export personal notes should have a warning

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#6360]
##### Detailed steps causing the problem:
- Open a cache list
- Select Menu - Export - Personal notes

##### Actual behavior after performing these steps:
As this option is under the export menu users might think, that they will be exported into a file (but not uploaded). This uploading might overwrite existing (and potentially newer) notes on the server.

##### Expected behavior after performing these steps:
Either we should make the functionality more clearer by renaming or replacing it (not under Export) or we should at least prompt a warning like "This will upload all personal notes of the caches in this list to the server and will overwrite existing personal notes on the server".

##### Version of c:geo used:
2017.02.23-NB1

**Comment by pstorch** [comment]
Yes, right. It is an upload, not an export. But a warning? Then we have to show a warning in the cache details, too. When uploading the personal note.

**Comment by Lineflyer** [comment]
The reason why I proposed the warning is, that a user tried out this menu item and failed to know what happend. After he finally learned, he lost server information as he expected to get an export file but in fact triggered an upload. 
In cache details it is named "Upload"..no warning needed. So depsite the many menu items we already have it might be better to have it one level up and rename it to "Upload all personal notes". In This case the meaning should be clear and no warning needed.

**Comment by pstorch** [comment]
Agreed.

**Comment by Lineflyer** [comment]
Renamed accordingly.

**Comment by caco3** [comment]
@Lineflyer Why do you call it "Mass" upload? if it is done from within a cache, I would expect that only the note of that cache gets uploaded!

**Comment by Lineflyer** [comment]
@caco3 
See procedure described in the initial post.
This issue is about uploading of notes from all caches in a list.

**Comment by caco3** [comment]
@Lineflyer A, I'm sorry, should have read it better :)
